controllers/v1/chat_message/index.js

```diff
@@ -0,0 +1,98 @@
+const { CREATED, UNAUTHORIZED, BAD_REQUEST, OK } = require("http-status-codes");
+
+//DB
+const ChatMessage = require('../../../model/v1/ChatMessage');
+
+const {
+  successWithData,
+  successNoData,
+} = require("../../../utils/successHandler");
+
+// Error
+const AppError = require("../../../utils/appError");
+
+exports.createMessage = async (req, res, next) => {
+  try {
+    const content = {
+      user: req.user.id,
+      message: req.body.message,
+      chat: req.body.chatId
+    }
+
```

Comment:
check if the body contains files, check out how it's done maybe on the Events controller!!